tests/dashboard.state.spec.js

```diff
@@ -34,6 +34,7 @@ describe('Dashboard', function() {
   describe('controller', function() {
     var controller;
```

Comment:
For this PR I tried limiting the test file changes to fix test failures due to 'strict' mode requirements. I was thinking we could change `var` to `let` holistically later on with a regex or code mod.